wg-manifests/charter.md

```diff
@@ -1,70 +1,50 @@
-- Provide a catalog of third-party apps for common services.
+We simply (automatically) synchronize the application and dependencies manifests to then elaborately combine (configure)them for full platform experience.
+Providing a consistent and tested end-to-end multi-tenant experience is the most important task of the platform/manifests WG.
+To achieve this we maintain an extensive testing suite that covers most basic scenarios users would expect from a Platform for ML orchestration.
+We also provide the documentation regarding, but not limited to installation, extension, security and architecture to enable users to run their own ML Platform on Kubernetes.
+Users may choose to derive from platform/manifests to create so called distributions, which are opinionated to satisfy individual requirements.
+Users may also choose to install individual components without the benefits of the platform.
 
 ### In scope
 
 #### Code, Binaries and Services
 
-- Maintain tooling to automate copying manifests from upstream app repos.
-- Maintain a catalog that will allow users to install Kubeflow apps and
-  common services easily on Kubernetes, either on the cloud or on-prem, without
-  depending on external cloud services or closed source solutions. Those
-  manifests are deployed using `kubectl` and `kustomize` and include:
-    1. A common set of manifests for the current official Kubeflow applications:
-        - Training Operators
-        - Kubeflow Pipelines (KFP)
-        - Notebooks
-        - KFServing
-        - Katib
-        - Central Dashboard
-        - Profile Controller
-        - PodDefaults Controller
-    1. Manifests for a set of specific common services:
-        - Istio
-        - KNative
-        - Dex
-        - Cert-Manager
-
-#### Cross-cutting and Externally Facing Processes
-
-##### With Application Owners
-
-- Aid applications owners in creating kustomize manifests for their application,
-  inside the app repo, if those don't exist already.
-- Communicate with application owners to agree upon the version they want to be
-  included in the next Kubeflow release.
-
-##### With Distribution Owners
-
-- Coordinate with distribution owners, to make sure they are in-sync about the
-  release schedule and have time to test and bring their distributions
-  up-to-date.
+- Enable users / distributions to install, extend and maintain Kubeflow as a end-to-end multi-tenant platform for multiple users
+- This includes dependencies, security efforts and exemplary integration with popular tools and frameworks.
+- Users can also install individual components without the benefits of the platform, but then they could also just directly fetch them from the WG releases.
+- Synchronize the manifests between working groups and make sure via integration tests that the components work end-to-end together as multi-tenant platform
+- Release tested releases of the Kubeflow platform for downstream consumption
```

Comment:
We need to clarify what is meant by "Kubeflow Platform", because this is not defined, or just not use that term.